Add support page (#77)
PrestaShop/open-source: content/support/_index.md

---
title: "Support"
---

> # I need help with PrestaShop

Author: ```suggestion
# Get help with PrestaShop
```

The PrestaShop open-source project can be freely downloaded and used by anyone, but it is provided "as is", with no warranty or dedicated support. 

Rest assured, there are many resources to help you if you need assistance.

## Community help

PrestaShop has a sizeable community, and there are many volunteers willing to help each other on the web:

* [Report a bug on GitHub]({{< devdocs "contribute/contribute-reporting-issues/" >}})
* [Join the community forums](https://www.prestashop.com/forums/)
* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/prestashop)
* [Discuss on Slack]({{< relref "/slack" >}})

## Contact a professional

> If you need serious help with your project, your best choice is to get contact a professional. There are thousands of agencies and freelance experts located around the globe:

Reviewer: "your best choice is to get contact a professional"

I think maybe there's a little typo here?

-> your best choice is to get in touch with a professional
-> your best choice is to contact a professional

Author: Good catch, thanks
```suggestion
If you need serious help with your project, your best choice is to contact a professional. There are thousands of agencies and freelance experts located around the globe:
```

> * [PrestaShop partner agencies and experts](https://www.prestashop.com/experts)

Reviewer 2: ```suggestion
```

Reviewer: I disagree with removing the link, but I think the link should be clear whether it's PrestaShop company or PrestaShop project.

```suggestion
* [PrestaShop SA partner agencies and experts](https://www.prestashop.com/experts)
```

Author: How about this?
```suggestion
* [List of agencies and experts](https://www.prestashop.com/experts) (sponsored by PrestaShop SA)
```

Reviewer: Better 👍

Reviewer 3: 404 error you need to add `/en/` but i also agree @Progi1984

* [Find a community expert on Slack]({{< relref "/slack" >}}) (within the `#jobs` channel)

## Learn PrestaShop

"Give a man a fish and you feed him for a day; teach a man to fish and you feed him for a lifetime."

> * [Training courses](https://www.prestashop.com/training)

Reviewer 2: ```suggestion
```

Reviewer: Similarly to what I said previously
```suggestion
* [PrestaShop SA training courses](https://www.prestashop.com/training)
```

Reviewer: Better 👍

> * [Training courses](https://www.prestashop.com/training)

Reviewer 3: 404 error you need to add `/en/` but i also agree @Progi1984
